dataflow/src/main/java/org/checkerframework/dataflow/cfg/block/Block.java

```diff
@@ -41,4 +42,11 @@
      * @return the predecessors of this basic block
      */
     Set<Block> getPredecessors();
+
+    /**
+     * Returns all the successors of this basic block.
```

Comment:
In contrast to `getPredecessors` there is an `all` in this sentence, which made me think this gives all transitive successors. It doesn't look like that is the case or intention.
So how about replacing all with `direct` or removing the `all`?